process-engine/charon
# ✨ Add fancy user profile to navbar

## What did you change?

This PR adds a better user profile to the navbar, currently the menu only supports login/logout but this can be extended in the future:

![okt -19-2017 13-30-17](https://user-images.githubusercontent.com/4148404/31809184-c4969a22-b576-11e7-9d4c-889c19f8a1b0.gif)

## How can others test the changes?

- Checkout branch
- run charon
- see login navbar section

## PR-Checklist

Please check the boxes in this list after submitting your PR:
